src/app.js

```diff
@@ -1,21 +1,62 @@
+/* eslint-disable no-console */
 'use strict';
 
+const http = require('http');
+
 /**
  * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
+*
+* Function takes 2 numbers and returns their sum
+*
+* sum(1, 2) === 3
+* sum(1, 11) === 12
+*
+* @param {number} a
+* @param {number} b
+*
+* @return {number}
+*/
+const PORT = process.env.PORT || 8080;
+
+function mainServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const { pathname, searchParams } = new URL(
+      req.url,
+      `http://${req.headers.host}`,
+    );
+
+    const parts = pathname.slice(1).split('/');
+
+    // logic if we have more than 2 same params
```

Comment:
Better to remove comments and write obvious code, or move part of code to variable with meaningful variable naming